models/eprop_iaf_psc_delta.cpp

```diff
@@ -207,7 +207,7 @@ nest::eprop_iaf_psc_delta::Parameters_::set( const DictionaryDatum& d, Node* nod
     throw BadProperty( "Firing rate regularization target rate f_target ≥ 0 required." );
   }
 
-  if ( kappa_ <= 0.0 or kappa_ > 1.0 )
+  if ( kappa_ < 0.0 or kappa_ > 1.0 )
   {
     throw BadProperty( "Eligibility trace low-pass filter from range [0, 1) required." );
```

Comment:
```suggestion
    throw BadProperty( "Eligibility trace low-pass filter from range (0, 1) required." );
```

Reply:
Sorry, my attention slipped and I mixed up the notation for closed interval (correct is `[`) and open (correct is `(`). Maybe we could keep both open, so that we can test the two edge cases.

Reply:
The notation is `[` for closed, i.e., `<=` or `>=`, and `(` for open, i.e., `>` or `<`, so the condition you suggest would correspond to `(0, 1]`. However, maybe we should keep it open into both directions, i.e., `0 < kappa, kappa > 1` and `(0,1)` to be able to test the edge cases.

Reply:
Please see the notation, e.g. here https://www.wikiwand.com/en/Interval_(mathematics)#Definitions_and_terminology

Reply:
@akorgor I don't really understand why you are still discussing this.  I do not understand the confusion.

Reply:
Sorry, just figured out the confusion - I was misreading the condition. I now understand that the comparison signs indicate what we discard, so yes correct is `kappa_ < 0.0` and `kappa > 1` implying `[0,1]`.